code/tasks/PopulateTimeZonesTask.php

```diff
@@ -33,26 +33,42 @@ class PopulateTimeZonesTask extends MigrationTask {
 	/**
 	 * {@inheritdoc}
 	 */
+	public function run($request) {
```

Comment:
This is now patched in the framework. See https://github.com/silverstripe/silverstripe-framework/pull/4313